client/state/selectors/is-connected-secondary-network-site.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isMainSiteOf } from 'state/selectors';
+import { isJetpackSiteSecondaryNetworkSite } from 'state/sites/selectors';
+
+/**
```

Comment:
Gotcha, thanks for clarifying. Indeed there is quite a difference between a connected secondary site, and a connected secondary site where the main network site is also connected.

So, I have a suggestion: let's make sure to describe what's the purpose of this selector. It can be confusing to understand the difference between it and `isJetpackSiteSecondaryNetworkSite` at first glance without looking at the code.